deploy/operator/gather.sh

```diff
@@ -17,7 +17,9 @@ function gather_hive_data() {
 }
 
 function gather_olm_data() {
-  oc get catalogsource "${ASSISTED_SERVICE_OPERATOR_CATALOG}" -n openshift-marketplace -o yaml > ${LOGS_DEST}/${ASSISTED_SERVICE_OPERATOR_CATALOG}.log
+  # We cant know if we're in disconnected mode, so we'll try both catalog names
```

Comment:
We can, if we'll add ``source /root/config`` before running this script in [operator-gather step](https://github.com/openshift/release/blob/master/ci-operator/step-registry/assisted/baremetal/operator/gather/assisted-baremetal-operator-gather-commands.sh#L58)

Reply:
I just don't want ``${LOGS_DEST}/${ASSISTED_SERVICE_OPERATOR_CATALOG}.log`` to get messy with an unrelated output like "could not find catalogsource ..."